bin/console

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "localize/ruby/client"
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+require "irb"
```

Comment:
What is it for?

Reply:
I see in Readme "You can also run `bin/console` for an interactive prompt that will allow you to experiment." I think it for checking Gen in console. If it unnecessary, I will remove it